### ISSUE TITLE
"default" not available on `PropertyType`

### ISSUE THREAD

**Issue by codeaid** [apioo/psx-schema#7]
It doesn't seem to possible to get the default value of a property. I have the following schema file:

```json
{
  "$schema": "http://json-schema.org/draft-04/schema#",
  "additionalProperties": false,
  "properties": {
    "quantity": {
      "type": "integer",
      "default": 1234,
      "minimum": 1
    }
  },
  "required": [
    "quantity"
  ],
  "type": "object"
}
```

My code is as follows:

```php
use PSX\Schema\Parser\JsonSchema;
use PSX\Schema\Schema;

$schema     = JsonSchema::fromFile($path);
$properties = $schema->getDefinition()->getProperties();

$quantity = $properties['quantity'];
$default  = $quantity->getDefault();

// $default is NULL
```

I started digging into the code and found the `\PSX\Schema\Parser\JsonSchema\Document` class, which has a method `getRecProperty`. Fetching defaults works if I add the following code to the method right after the `enum` property population:

```php
if (isset($data['default'])) {
    $property->setDefault($data['default']);
}
```

Not sure if that's the right solution but I can't see the `1234` value (default) being set anywhere if I dump the `PropertyType` instance.

**Comment by chriskapp** [comment]
Hi, yes this property is missing. It was probably not noticed since it is a meta property which does not apply any validation rules but nonetheless we should add this. I think your code is correct. If you like you can make a pull request otherwise I can also simply add this.

**Comment by chriskapp** [comment]
This is fixed in the latest version see #8
